shared/chat/payments/confirm/index.js

```diff
@@ -109,16 +109,9 @@ const PaymentsConfirm = (props: Props) => (
           ))}
         </Kb.ScrollView>
         <Kb.ButtonBar align="center" direction="row" fullWidth={true} style={styles.buttonBar}>
-          <Kb.Button
-            type="Secondary"
-            small={Styles.isMobile}
-            onClick={props.onCancel}
-            style={styles.cancelButton}
-            label="Cancel"
-          />
+          <Kb.Button type="Secondary" onClick={props.onCancel} style={styles.cancelButton} label="Cancel" />
```

Comment:
should this be waitingbutton?

Reply:
It is immediate, no network or anything in play.